src/AccountManager.mo

```diff
@@ -360,40 +365,41 @@ module {
     /// This method allows a user to deposit tokens by setting up an allowance on their account with the service
     /// principal as the spender and then calling this method to transfer the allowed tokens.
     public func depositFromAllowance(p : Principal, account : ICRC1.Account, amount : Nat) : async* DepositFromAllowanceResponse {
-      if (amount < minimum(#deposit)) return #err(#TooLowQuantity);
+      if (amount <= ledgerFee_) {
+        log(p, #depositViaAllowanceError(#TooLowQuantity));
+        return #err(#TooLowQuantity);
+      };
 
       let transferResult = await* processAllowance(p, account, amount);
 
-      let originalCredit : Nat = amount - fee(#deposit);
+      let originalCredit : Nat = amount - ledgerFee_;
```

Comment:
According to the spec here https://github.com/dfinity/ICRC-1/tree/main/standards/ICRC-2#icrc2_transfer_from  `transfer_from` adds the ledger fee to the `amount` value so that the `to` account gets exactly `amount. Therefore here `originalCredit = amount`. No subtraction of `ledgerFee_`.